modules/llrt_buffer/src/buffer.rs

```diff
@@ -332,10 +333,18 @@ impl ModuleDef for BufferModule {
         let globals = ctx.globals();
         let buf: Constructor = globals.get(stringify!(Buffer))?;
 
+        let constants = Object::new(ctx.clone())?;
+        #[cfg(target_pointer_width = "32")]
+        constants.set("MAX_LENGTH", (1u32 << 30) - 1);
+        #[cfg(target_pointer_width = "64")]
+        constants.set("MAX_LENGTH", (1u64 << 53) - 1)?;
```

Comment:
I think you can do usize::MAX and get rid of that?

Reply:
Thanks for pointing that out. And I will comment here that I like very much the modularization you have implemented. :)